src/input-mappings.js

```diff
@@ -67,20 +72,22 @@ const config = {
         surfacetouchend: "thumb_up",
         thumbsticktouchstart: "thumb_down",
         thumbsticktouchend: "thumb_up",
-        triggerdown: ["action_teleport_down", "index_down"],
-        triggerup: ["action_teleport_up", "index_up"],
+        triggerdown: "index_down",
+        triggerup: "index_up",
         "axismove.reverseY": { left: "move" },
-        right_dpad_east: "snap_rotate_right",
-        right_dpad_west: "snap_rotate_left",
         abuttondown: "action_teleport_down",
         abuttonup: "action_teleport_up"
       },
       "daydream-controls": {
-        trackpaddown: "action_teleport_down",
+        trackpad_dpad4_pressed_west_down: "snap_rotate_left",
+        trackpad_dpad4_pressed_east_down: "snap_rotate_right",
+        trackpad_dpad4_pressed_center_down: "action_teleport_down",
         trackpadup: "action_teleport_up"
       },
       "gearvr-controls": {
-        trackpaddown: "action_teleport_down",
+        trackpad_dpad4_pressed_west_down: "snap_rotate_left",
+        trackpad_dpad4_pressed_east_down: "snap_rotate_right",
+        trackpad_dpad4_pressed_center_down: "action_teleport_down",
```

Comment:
Snap rotate works well on gearvr, but it feels odd that up and down are dead. We should map them to move forward/back or at least map them to teleport.

Reply:
Ok, I think I will map them to teleport for all the trackpad platforms. We can implement a sort of blink-forward if it makes sense later.